app/models/user.rb

```diff
@@ -12,11 +12,7 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
 
   def has_pending_invitation?
-    if self.invited_organization_id && self.membership.nil?
-      true
-    else
-      false
-    end
+    return !self.invited_organization_id.nil?
```

Comment:
Was previously checking to see if there was no membership record and if there was an invited organization id (not nil).  
Now that members can be invited to join different organizations, I no longer want to check for existing membership.  
The only factor in having a pending invite is the invited_organization_id, which will be nil when there is no invitation and will be set to nil when the invitation is accepted or denied.